integration-tests/part-filename/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>quarkus-openapi-generator-integration-tests</artifactId>
+    <groupId>io.quarkiverse.openapi.generator</groupId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>quarkus-openapi-generator-it-part-filename</artifactId>
+  <name>Quarkus - Openapi Generator - Integration Tests - PartFilename</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkiverse.openapi.generator</groupId>
+      <artifactId>quarkus-openapi-generator</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-multipart-provider</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-reactive-common</artifactId>
+    </dependency>
```

Comment:
We declare dependencies that are specific for RESTEasy Reactive or Classic in its equivalent profile in [integration-tests/pom.xml](https://github.com/quarkiverse/quarkus-openapi-generator/blob/5f10fa319d4b58d37ce68e6aaa953796241c11a3/integration-tests/pom.xml#L47). I'm not sure about `resteasy-multipart-provider`, but you can move at least `quarkus-resteasy-reactive-common` to there.

Reply:
Thanks for the info, I noticed it, but I did not want to put them there, as they're (only) relevant to the `part-filename` tests and I didn't want to influence the rest of the tests.

I need them to be able to compile the tests as they're checking both types of annotations and I couldn't figure out, how to do it in a more elegant way. Or maybe I'm just doing something wrong?

Btw. I think the `quarkus-resteasy-reactive-common` should already be part of `quarkus-rest-client-reactive-jackson` which you already have there.